main/integration-tests/support/resources/workspace-types/workspace-types.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-shadow */
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const _ = require('lodash');
+
+const CollectionResource = require('../base/collection-resource');
+const WorkspaceType = require('./workspace-type');
+
+class WorkspaceTypes extends CollectionResource {
+  constructor({ clientSession }) {
+    super({
+      clientSession,
+      type: 'workspace-types',
+      childType: 'workspace-type',
```

Comment:
I think we have been doing camel case for the types per the second tab in the spreadsheet

Reply:
Done